tao_compiler/mlir/disc/transforms/fusion_utils.cc

```diff
@@ -552,7 +552,7 @@ FusionPatternBase::FusionPatternBase(SmallVectorImpl<Operation*>& op_list)
 // fusion pattern contains.
 int FusionPatternBase::effectiveSize() {
   return llvm::count_if(
-      op_list_, [](Operation* op) { return !matchPattern(op, m_Constant()); });
+      op_list_, [](Operation* op) { return !isa<lmhlo::ConstantOp>(op); });
```

Comment:
why is it narrowed to `lmhlo::ConstantOp`?

Reply:
m_Constant only works for const on tensor level.